src/graphics/render_wavefront.cpp

```diff
@@ -234,32 +233,32 @@ Writes Wavefront object file that defines the material
 			}
 			else
 			{
-				fprintf(file,"usemtl %s\n",Graphical_material_name(material));
+                wavefront_output << "usemtl " << Graphical_material_name(material) << std::endl;
```

Comment:
I suggest adding a method to cmzn_material:
```
const char* getName() const
{
    return this->name;
}
```
I don't think the null name happens any more, but you may handle that case in the above method if feeling pessimistic.